services/ingestion/helpers/getCountryCode.js

```diff
@@ -0,0 +1,14 @@
+// Helper to get country code based on few exceptions
```

Comment:
To be clear, here we make the conversion NUTS0 -> ISO_3166-2 😉

Reply:
Added these details :)